connector/aws-lambda/THIRD_PARTY_NOTICES

```diff
@@ -0,0 +1,22 @@
+Camunda AWS Lambda Connector
```

Comment:
We are missing the THIRD_PARTY_NOTICE GitHub workflow for the Connectors. Question to be answered here: Do we want to have one file for all Connectors or one per Connector? I think the latter would be nice but we need to be able to handle this in a GitHub workflow as well.

Reply:
Eventually I came up with the following configuration (see the new workflow):

- First per-connector THIRD_PARTY_NOTICES files are generated 
- Then an aggregate THIRD_PARTY_NOTICES file is generated for the whole project. This one then gets uploaded with the GitHub release.

Reply:
Cool, I'll have a look 👍

Reply:
Looks good to me. It's a bit sad that we lost the header for each individual Connector but it's also fine, I think. Having both individual files to link to and an overall file to distribute is nice, I think. 👍 

Can you create a follow-up to adjust the public docs at https://docs.camunda.io/docs/reference/dependencies once we're done here? We need to adjust the links to point to the repository here. I think we can remove the individual Connectors from that page even, linking only to the overall file and hinting to look at individual files in the subdirectories.

Reply:
https://github.com/camunda/team-connectors/issues/256